### PR TITLE
Add new flags for kruize local and logging

### PR DESCRIPTION
**Pull request by khansaad**
## PR Title :boom:

This PR adds new flags in the `kruize-clowdapp.yaml`

## Why do we need this change? :thought_balloon:

To enable `kruize-local` use case and for logging request and response for debugging purpose.

## Documentation update? :memo:

- [ ] Yes
- [x] No

## Security Checklist :lock:

Upon raising this PR please go through [RedHatInsights/secure-coding-checklist](https://github.com/RedHatInsights/secure-coding-checklist)

## :guardsman: Checklist :dart:

- [ ] Does this change depend on specific version of Kruize
  - If yes what is the version no: 0.0.21
  - [ ] Is that image available in production or needs deployment? 
     Yet to be released
- [ ] Bugfix
- [ ] New Feature
- [ ] Refactor
- [ ] Unittests Added
- [ ] DRY code
- [ ] Dependency Added
- [ ] DB Migration Added

## Additional :mega:

Feel free to add any other relevant details such as __links, notes, screenshots__, here.

### CODE REVIEW

File: kruize-clowdapp.yaml [hunk 1]
```diff
@@ -73,6 +73,10 @@ objects:
             value: "prometheus-k8s"
           - name: savetodb
             value: "true"
+          - name: local
+            value: "false"
+          - name: logAllHttpReqAndResp
+            value: "true"
```
Comment (saltgen) [inline kruize-clowdapp.yaml:79]: @khansaad "false" for `logAllHttpReqAndResp` makes sense here as well, we can override it on EE when required, also thoughts on switching to upper case?

**Review by saltgen — CHANGES_REQUESTED** [review]

File: kruize-clowdapp.yaml (hunk 1 above)
Comment (khansaad) [reply kruize-clowdapp.yaml:79]: @saltgen We must debug the reasons for the failure of the `updateRecommendation` calls. This flag needs to be set to `true` until the issue is resolved.

Switched the variable to upper case as suggested.

File: kruize-clowdapp.yaml [hunk 2]
```diff
@@ -130,6 +134,10 @@ objects:
               value: "prometheus-k8s"
             - name: savetodb
               value: "true"
+            - name: local
+              value: "false"
+            - name: LOG_ALL_HTTP_REQ_AND_RESPONSE
+              value: "true"
```
Comment (saltgen) [inline kruize-clowdapp.yaml:140]: @khansaad I don't think we need these vars here, i.e. for the type job.

File: kruize-clowdapp.yaml [hunk 3]
```diff
@@ -183,6 +191,10 @@ objects:
...
```
Comment (saltgen) [inline kruize-clowdapp.yaml:197]: Ditto

**Review by saltgen — CHANGES_REQUESTED** [review]

File: kruize-clowdapp.yaml (hunk 1 above)
Comment (saltgen) [reply kruize-clowdapp.yaml:79]: Ah okay, it seemed like `logAllHttpReqAndResp` was for kruize local.

File: kruize-clowdapp.yaml (hunk 2 above)
Comment (khansaad) [reply kruize-clowdapp.yaml]: Updated now.

**Review by saltgen — APPROVED** [review]
Thanks @khansaad LGTM!

**Pull request merged**